features/step_definitions/vendor_steps.rb

```diff
@@ -99,6 +99,10 @@
   expect(div).not_to have_selector("input[value='#{FactoryBot.attributes_for(:original_ownership)[:name]}']")
 end
 
+Then /the vendor should have the other vendor's name/ do
```

Comment:
What's the purpose behind this step?

Reply:
Is this for a duplicate vendor name? What is the "other vendor's name"?

Reply:
Yes. The "other vendor" steps are only used for the duplicate vendor test. When I try to change the vendor's name to duplicate another vendor's, the update should not succeed. However, since I want the edits to still be reflected on the form, the name filled in should be the duplicate name.